Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -680,8 +794,12 @@ public function setCode($code)
             $this[$index] = new Token($token);
         }
 
+        $transformers = Transformers::create();
+        $transformers->transform($this);
```

Comment:
@GrahamCampbell this is a critical bug fix in case Transformers are already in 1.4, because it would mean that we sometimes parse PHP code without transforming tokens, and fixers are totally broken if they run on a collection of untransformed tokens (otherwise we would not have introduced transformers to fix them)